mathsnuggets/widgets/taylor.py

```diff
@@ -0,0 +1,19 @@
+import sympy
+
+from mathsnuggets.core import fields, form
+
+test = {"function": "x sin x"}
+
+
+class Taylor(form.Form):
+    """Taylor Expansion"""
+
+    template = """Expand `expression``expand`"""
```

Comment:
There are no fields called `expand`. You called it taylor

Reply:
Perhaps you could change it to

    Taylor expansion of `expression` of order `order` with respect to `x` around `x0`

Reply:
Okay Sir, its because I didn´t know the format of template so I just copied it from the other expansion class.